{
	"deprecated": false,
	"cpeName": "cpe:2.3:a:ibm:planning_analytics_local:2.0.1:*:*:*:*:*:*:*",
	"cpeNameId": "E0367558-26C7-4713-AF70-03E1A17EC8C0",
	"created": "2019-10-03T13:42:48.470",
	"lastModified": "2019-10-03T13:42:48.470",
	"titles": [
		{
			"title": "IBM Planning Analytics Local 2.0.1",
			"lang": "en"
		}
	],
	"refs": [
		{
			"ref": "https://www.ibm.com/mysupport/s/topic/0TO500000002PWWGA2/planning-analytics-local?language=en_US&productId=01t50000004Xdyn",
			"type": "Product"
		},
		{
			"ref": "https://exchange.xforce.ibmcloud.com/vulnerabilities/145118",
			"type": "Advisory"
		}
	]
}
